verification/latency_test/latency_testing.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
```

Comment:
I think when the code in the notebook is pretty complete, it'll probably make sense to move most of it in here so it can be run from the command line. If most of the functionality is inside a function or a set of functions, you should be able to import them in the notebook if you want to be able to run them there without having to maintain two copies of the code.

Then, you can have a little section at the end that makes it possible for this to be run as an executable. See `module/fake.py` for an example of this kind of thing.